src/constructs/iam/policies/s3-get-object.ts

```diff
@@ -1,23 +1,31 @@
+import type { GuPrivateS3ConfigurationProps } from "../../../utils/ec2";
 import type { GuStack } from "../../core";
 import { GuDistributionBucketParameter } from "../../core";
 import type { GuNoStatementsPolicyProps } from "./base-policy";
 import { GuAllowPolicy } from "./base-policy";
 
 export interface GuGetS3ObjectPolicyProps extends GuNoStatementsPolicyProps {
   bucketName: string;
-  path?: string;
+  paths?: string[];
 }
 
-export class GuGetS3ObjectPolicy extends GuAllowPolicy {
+export class GuGetS3ObjectsPolicy extends GuAllowPolicy {
```

Comment:
I think this rename warrants a minor version bump? The current PR title (`refactor: `) will be a no-op IIUC.

Reply:
Good point. Is it best to add `!`? I think that would be a major version bump, is that what we want here?

Reply:
Reading the [docs](https://github.com/semantic-release/commit-analyzer) again, it suggests `refactor: ` is actually ok and what we'd need to do is add `BREAKING CHANGE` to the footer?

From https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines:
> Footer
> The footer should contain any information about Breaking Changes and is also the place to reference GitHub issues that this commit closes.
>
> Breaking Changes should start with the word BREAKING CHANGE: with a space or two newlines. The rest of the commit message is then used for this.

A major version bump sounds correct. From https://semver.org/:

> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.

I think this'll amount to v0.41.0 after reading [this item](https://semver.org/#spec-item-4) in the spec, rather than v1.0.0?

I find this all a bit confusing, but this is likely because I've never written structured commits!

Reply:
More confusion: I think adding `!` is valid for the [PR title linter](https://github.com/amannn/action-semantic-pull-request#validation) to describe a breaking change.

I think I'm going to end up referring to something like [this guide](https://www.conventionalcommits.org/en/v1.0.0/) quite often!

Reply:
> I find this all a bit confusing

I'm glad it's not just me 😅 

> I think this'll amount to v0.41.0 after reading [this item](https://semver.org/#spec-item-4) in the spec, rather than v1.0.0?

Ah okay cool, I'd missed that part. I've merged now (with `!`) so let's hope we've figured this out correctly 🤞